pkg/operator/ceph/object/bucket.go

```diff
@@ -140,7 +148,17 @@ func getBucketMetadata(c *Context, bucket string) (*ObjectBucketMetadata, bool,
 		return nil, false, errors.Wrapf(err, "failed to read buckets list result=%s", result)
 	}
 
-	createdAt, err := time.Parse("2006-01-02 15:04:05.999999999Z", s.Data.CreationTime)
+	timeParser := beforeOctopusTime
+	version, err := cephver.ExtractCephVersion(opcontroller.OperatorCephBaseImageVersion)
+	if err != nil {
+		logger.Errorf("failed to extract ceph version. %v", err)
+	}
+
+	vv := *version
```

Comment:
If the version failed to be extracted, this will be nil and crash, right? In that case shall we default to assume it's octopus? This seems like a safe enough assumption instead of always failing the bucket request.